spec/models/card_spec.rb

```diff
@@ -1,111 +1,60 @@
 require 'rails_helper'
 
-  describe Card do
-    let!(:card) { build(:card) }
-    let!(:wrong_card) { Card.create(original_text: 'Привет',
-                                   translated_text: 'ПривЕТ') }
-
-    context '#words_cannot_be_equal' do
-
-      it 'does not pass the same words' do
-        expect(wrong_card).not_to be_valid
-      end
+describe Card do
+  let!(:user) { create(:user) }
+  let!(:deck) { create(:deck, user_id: user.id) }
+  let!(:card) { create(:card, deck_id: deck.id) }
+  let!(:wrong_card) { Card.create(original_text: 'Привет',
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.